### ISSUE TITLE
Installing/Training on Google Colab

### ISSUE THREAD

**Issue by Overload119** [ludwig-ai/ludwig#2874]
**Describe the bug**

**To Reproduce**

1. Create a Jupyter notebook and run any training task.

```
!pip install ludwig
!ludwig train --config '/content/ludwig-config.yml' --dataset '/content/data.csv'
```

The task stops with error:

<details>
<summary>Error Message</summary>

```
This program was compiled against version 3.9.2 of the Protocol Buffer runtime library, which is not compatible with the installed version (3.20.0).  Contact the program author for an update.  If you compiled the program yourself, make sure that your headers are from the same version of Protocol Buffers as your link-time library.  (Version verification failed in "bazel-out/k8-opt/bin/tensorflow/core/framework/tensor_shape.pb.cc".)
```

</details>

I've ran `!pip install protobuf==3.9.2` but that leads to another error:

`AttributeError: 'NoneType' object has no attribute 'message_types_by_name`

**Expected behavior**
It starts to train without errors.

**Environment (please complete the following information):**

- OS: Linux
- Version: Google Python Compute
- Python version 3.7
- Ludwig version 0.6.4

**Additional context**

If I run this block, I can workaround the issue to run train.

```
!pip uninstall tensorflow
```

**Comment by justinxzhao** [comment]
Hi @Overload119,

This is a known incompatibility between Ludwig on PyTorch and Google Colab. Not totally sure why this is happening, and it's also frustrating that the dependency issue comes up at runtime.

However, the workaround I've found is to first uninstall tensorflow, as you've mentioned in your comment.

```
!pip uninstall tensorflow
```

We've updated our sample notebooks to reflect this as well.
https://ludwig.ai/latest/examples/text_classification/
